skt/executable.py

```diff
@@ -218,6 +256,18 @@ def cmd_build(cfg):
     tstamp = datetime.datetime.strftime(datetime.datetime.now(),
                                         "%Y%m%d%H%M%S")
 
+    # Remove existing results from previous runs (if any)
+    for filename in os.listdir(cfg.get('workdir')):
+        if filename.startswith('build.'):
+            try:
+                os.unlink(os.path.join(cfg.get('workdir'), filename))
+            except OSError:
+                pass
+
```

Comment:
I understand this is a safety thing, but have we seen or expect leftover files?  Doesn't each run have its own directory, meaning this can/should never happen?

Reply:
This is more for running things by hand, not from pipeline. In the pipeline, new directories should be created for the runs. But if people (eg. the developers, or us when testing things out) would like to be able to reproduce the runs, they probably won't bother with creating new directories and deleting the old files. The result and report files would be overwritten anyways, this just makes it explicit that you shouldn't rely on the old output if you decide to use the same directory.

Reply:
Ah, ok.  That makes sense then.  Thanks!